src/controllers/Pwd-Reset-Controller.coffee

```diff
@@ -74,25 +74,31 @@ class Pwd_Reset_Controller
 
     #Validating token
     if (token == null or username == null or username is '' or token is '')
+      @.errorMessage = 'Token is invalid'
       return @.render_Page @.jade_password_reset_fail, {errorMessage: 'Token is invalid'}
```

Comment:
in this case we should also use the @.erroeMessage value in the call to @.jade_password_reset_fail